### ISSUE TITLE
There was a problem with the request.

### ISSUE THREAD

**Issue by kvlahromei** [hotosm/oam-uploader#60]
Hi,

I tried to submit an filled form, but only received an error notification. The POST XHR looks like
```javascript 
{"uploader":{"name":"Hansestadt Rostock – Kataster-, Vermessungs- und Liegenschaftsamt","email":"geodienste@rostock.de"},"scenes":[{"contact":{"name":"Hansestadt Rostock – Kataster-, Vermessungs- und Liegenschaftsamt","email":"geodienste@rostock.de"},"title":"Luftbild Rostock und Umgebung 2000","provider":"Hansestadt Rostock – Kataster-, Vermessungs- und Liegenschaftsamt","platform":"aircraft","sensor":"unknown","acquisition_start":"2000-10-31T23:00:00.000Z","acquisition_end":"2000-11-30T23:00:00.000Z","urls":["https://geo.sv.rostock.de/download/opendata/luftbild_2000/luftbild_2000.ecw.zip"]}]}
```

> POST /uploads?access_token=xyz
> HTTP/1.1
> Host: upload-api.openaerialmap.org
> User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:51.0) Gecko/20100101 Firefox/51.0
> Accept: */*
> Accept-Language: de,en-US;q=0.7,en;q=0.3
> Accept-Encoding: gzip, deflate, br
> Content-Type: application/json
> Referer: https://upload.openaerialmap.org/
> Content-Length: 586
> Origin: https://upload.openaerialmap.org
> DNT: 1
> Connection: keep-alive

It returns an HTTP 400. Anything wrong on my input?
(I disabled adblockers etc. but we have an firewall proxy)

**Comment by smit1678** [comment]
@kvlahromei Huh, looks like we have: 

```
{"name":"ValidationError","details":[{"message":"\"name\" length must be less than or equal to 30 characters long"
```

Maybe we can do a shorter name? 

We probably could allow longer names but need to think through the impact.

**Comment by kvlahromei** [comment]
Not a problem at all, but IMHO it would be worthwile to check / communicate this kind of limitations back to the frontend. Otherwise the enduser doesn't know what's wrong 😉

**Comment by smit1678** [comment]
yes, 100% agreed. Noted and will make sure this is incorporated. Thanks for this feedback.

**Comment by smit1678** [comment]
@kvlahromei Just saw other details from your upload. It looks like you're trying to upload a zip file. You'll need to upload a file that is unzipped.

**Comment by kvlahromei** [comment]
Well ok, i can strip the archive off. On the other hand, I guess compressing aerial imagery dumps is pretty common. So an notification on supported image formats, or an unzip call (be aware on zip-bombs 💥 ) would be very handy to make support a tip easier ^^

**Comment by smit1678** [comment]
Agreed! Fwiw, we also convert into a compressed geotiff for final storage. 

We're working on providing additional and better documentation across all the components. If you'd like to help contribute, the frontend code for the Uploader is here: https://github.com/hotosm/oam-uploader. Please open some issues and then we can work on that together.

**Comment by kvlahromei** [comment]
Thx for your feedback.

Unfortunatly I'm not familar with node.js from an dev perspective, but I'm happy, if our city can contribute some more DOPs 😄
